.github/workflows/terragrunt-plan-staging.yml

```diff
@@ -25,6 +25,9 @@ env:
   TF_VAR_rds_db_password: ${{ secrets.STAGING_DB_PASSWORD }}
   TF_VAR_slack_webhook: ${{ secrets.STAGING_SLACK_WEBHOOK }}
   TF_VAR_gc_notify_callback_bearer_token: ${{ secrets.STAGING_GC_NOTIFY_CALLBACK_BEARER_TOKEN }}
+  TF_VAR_gc_notify_url: https://staging.notification.cdssandbox.xyz
```

Comment:
Were still using the production instance of notify ?

Reply:
We'll be using the new instance of GC notify for staging. Each environment has its own service.